modules/system-management/pages/system-metrics.adoc

```diff
@@ -0,0 +1,102 @@
+= Metrics Reporting
+:description: Overview of metrics gathering in TigerGraph Server.
+
+You can use `gadmin metric` commands to collect the following metrics related to state of resource usage on your instances:
+
+* CPU usage
+* Memory usage
+* Disk space usage
+* Network-related information
+
+For a full list of options available to `gadmin metric`, run `gadmin metric -h` in the command line to view the help text for the command.
+
+== Report CPU usage
+
+Use the command `gadmin metric -t cpu` to report CPU usage by different TigerGraph services on a cluster or a node in the cluster.
+
+To report CPU usage on select nodes, use the `--host` flag or the `-m` shorthand, followed by the nodes for which you want CPU usage data, separated by commas.
+If you choose not to specify nodes, then the command reports CPU usage for all nodes in the cluster.
+
+To select specific services to report data for, specify the service IDs after the command.
+If you don't specify a service, the command reports data for all services.
+
+NOTE: Service ids currently include the following:
+`gpe gse restpp gsql ifm gui ctrl kafka etcd zk nginx ts3 ts3serv dict admin kafkaconn kafkastrm-ll`
+
+The following command reports CPU usage by the `GUI` service for m1 and m2:
+
+[.wrap,console]
+----
+$ gadmin metric gui -t cpu -m m1,m2
+----
+
+The following command reports CPU usage by all services for m1 and m2:
+
+[.wrap,console]
+----
+$ gadmin metric gui -t cpu -m m1,m2
+----
+
```

Comment:
Do we have a master table of all gadmin commands and their function? If so, we need to add a link to it. If not, then we should consider for future work.

Reply:
A master list of all the commands are here: https://docs.tigergraph.com/tigergraph-server/current/system-management/management-commands

Though some of these are likely out-of-date, and the formatting could be improved as well.